<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchStudent</class>
 <widget class="QMainWindow" name="SearchStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>721</width>
      <height>491</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#606368;

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#0C34F0;
font: 75 16pt bold &quot;Arial Nova&quot;;</string>
     </property>
     <property name="text">
      <string>Recherche d'Etudiant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#202124;
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>177</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton1">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
</string>
        </property>
        <property name="text">
         <string>Par CIN :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;
</string>
        </property>
        <property name="text">
         <string>Par Nom et Pr√©nom : </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:#3773FE;
color:white;
font: 75 12pt &quot;Arial&quot;;
border-radius:10px;
}

QPushButton:hover:!pressed
{
  border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>531</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
color:black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#202124;
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#202124;
color: rgb(255, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
